src/server/index.js

```diff
@@ -47,6 +47,9 @@ function buildNavigationRouteRegexp() {
 // Configure the Keycloak client.
 const keycloakClient = authn.getKeycloakClient();
 
+const nextI18next = require("../i18n");
+const nextI18NextMiddleware = require("next-i18next/middleware").default;
```

Comment:
Can we use `import` syntax here?
```suggestion
import nextI18next from "../i18n";
import nextI18NextMiddleware from "next-i18next/middleware";
```
Except these lines would then have to be moved to the top of the file with the rest.

Reply:
This worked.